hera_mc/cm_utils.py

```diff
@@ -297,6 +297,18 @@ def html_table(headers, table):
     return s
 
 
+def csv_table(headers, table):
```

Comment:
this function needs a docsting. Also, please do not use single letter variable names.